inference-engine/include/ie_iextension.h

```diff
@@ -147,6 +147,7 @@ class INFERENCE_ENGINE_API_CLASS(ILayerExecImpl) : public ILayerImpl {
 
 /**
  * @deprecated Implement IExtension::getImplTypes and IExtension::getImplementation
+ * The interface will be removed in 2021.1 release.
```

Comment:
Why didn't you add this message to `INFERENCE_ENGINE_DEPRECATED` macros?

Reply:
some lines very long and adding new messages leads to code style issues, while splitting into 2 separate lines leads to doxygen issues :)

Reply:
does it sound good?)

Reply:
Yes, sorry. I thought that I sent a comment. As a variant we can use comment `// NOLINT` to disable code style check for current line.